web/src/app/modules/shared/components/presentation/tabs/tabs.component.html

```diff
@@ -1,10 +1,12 @@
 <div class="clr-row clr-align-items-center clr-justify-content-between">
-  <div class="clr-col-10">
+  <div class="clr-col-6">
     <app-breadcrumb [path]="title"> </app-breadcrumb>
   </div>
-  <div class="clr-col-2">
-    <div *ngIf="buttonGroup" class="action-buttons">
-      <app-button-group [view]="buttonGroup"></app-button-group>
+  <div class="clr-col-6">
+    <div *ngIf="titleComponents?.length > 0" class="title-container">
+      <div *ngFor="let component of titleComponents; let i = index; trackBy: identifyPath" class="item">
```

Comment:
`identifyPath` is a unique trackbyFn for breadcrumbs.

Remove `let i = index` and change the trackByFn to `trackByIndex` via `import trackByIndex from 'src/app/util/trackBy/trackByIndex';`. Without this set correctly, components passed through this method will update with each new content response which can be problematic for any persisting UI states (e.g. opened modals)